src/sentry/integrations/github/issues.py

```diff
@@ -72,10 +72,11 @@ def create_default_repo_choice(self, default_repo: str) -> tuple[str, str]:
 
     def get_create_issue_config(
         self, group: Group, user: User, **kwargs: Any
-    ) -> Sequence[Mapping[str, Any]]:
+    ) -> List[Dict[str, Any]]:
```

Comment:
I typed a lot of stuff. For this change, I found out it's best practice to type func arguments as abstract and return types as concrete